README.pod

```diff
@@ -0,0 +1,34 @@
+=encoding utf-8
+
```

Comment:
Don't really get this in Pod format; but it does not really matter. It's legible by GitHub, so I guess it's as good as markdown or anything like that.